ansible/site.yml

```diff
@@ -103,3 +103,8 @@
   roles:
     - role: uan_interfaces
       uan_can_setup: yes
+      when:
+        - not cray_cfs_image|default(false)|bool
+        - uan_set_compute_default_route
```

Comment:
Could this be controlled by `uan_can_setup` instead? In can be set independently for UAN hosts and Compute hosts. That allow us to not introduce another toggle.

Reply:
Yes, it can!  I will make it so.